src/test/java/com/wikia/webdriver/testcases/adstests/TestAmazonAds.java

```diff
@@ -61,10 +35,10 @@ private void testAmazonAd(String wikiName, String path, boolean debugMode) {
     if (debugMode) {
       testedPage = urlBuilder.appendQueryStringToURL(testedPage, "amzn_debug_mode=1");
     }
-    AdsAmazonObject amazonAds = new AdsAmazonObject(driver, testedPage, networkTrafficInterceptor);
+    AdsAmazonObject amazonAds = new AdsAmazonObject(driver, testedPage);
 
     amazonAds.verifyAmazonScriptIncluded();
-    amazonAds.verifyCallToAmazonIssued();
+//    amazonAds.verifyCallToAmazonIssued();
```

Comment:
I would like to get rid of logOut call since it will decrease the execution time of the tests. So, most obvious solution for me was creation a new test template or removing logOut call for all tests. I chose first one. :wink: Your suggestions how to get rid of logOut call?